### PR TITLE
fix sphinx errors / warnings

### PR DESCRIPTION
**Pull request by andre-merzky**
this should fix the RTD build

### CODE REVIEW

File: src/radical/utils/dict_mixin.py [hunk 1]
```diff
@@ -360,14 +360,15 @@ def iter_diff(a, b):
 #
 def dict_diff(a, b):
     '''
-    return a dict of the form:
+    return a dict of the form::
```
Comment (AymenFJA) [inline src/radical/utils/dict_mixin.py:363]: Why double colon `::`?

**Review by AymenFJA — APPROVED** [review]
Thanks, LGTM

**Comment by andre-merzky** [comment]
@AymenFJA , @kartikmodi : this is ready for review now.

**Comment by andre-merzky** [comment]
Oh, dammit - I pressed merge in a wrong PR.  @AymenFJA : please have a look at the diff anyway and let me know if you think anything needs addressing!

**Pull request merged**